docs/learn/learn-nominator.md

```diff
@@ -48,15 +48,43 @@ nomination. However, the election algorithm attempts to minimize this situation,
 occur often, so you should almost always see only a single active nomination per era. See the
 [section on Phragmén optimization](learn-phragmen.md#optimizations) for more details.
 
+### Staking Election Stages
+
+The staking election system has 3 stages for both validators and nominators, namely "intention", 
+"electable/electing", and "active".
+
+- **intention to nominate:** an account that has stated the intention to nominate; also called simply 
+a "nominator".
+- **electing nominator:** a nominator who is selected to be a part of the input to the NPoS election 
+algorithm. This selection is based on stake, and is done using the bags-list pallet.
```

Comment:
algorithm could link to the phragmen page, 
bags-list could link to its rustdoc?